project/MimaExcludes.scala

```diff
@@ -34,6 +34,10 @@ import com.typesafe.tools.mima.core.ProblemFilters._
  */
 object MimaExcludes {
 
+  // Exclude rules for 3.3.x
+  lazy val v33excludes = v32excludes ++ Seq(
```

Comment:
We need not address there here, but I'm wondering if we should carry forward all the previous excludes? In master (currently 3.3.0) we're only concerned with excluding changes since 3.2.x. If, somehow, a previous exclusion matched a new, legitimate breaking change of the same type, we'd be masking it, no? or maybe i'm not thinking about it correctly.